src/Tests/Microsoft.NET.Sdk.Razor.Tests/ScopedCssIntegrationTests.cs

```diff
@@ -78,16 +78,13 @@ public void CanOverrideScopeIdentifiers()
             File.Move(Path.Combine(projectDirectory.Path, "Components", "Pages", "Counter.razor.css"), styles);
 
             var build = new BuildCommand(projectDirectory);
-            build.Execute("/p:EnableDefaultScopedCssItems=false", "/p:_RazorSourceGeneratorWriteGeneratedOutput=true").Should().Pass();
+            build.Execute("/p:EnableDefaultScopedCssItems=false").Should().Pass();
 
             var intermediateOutputPath = Path.Combine(build.GetBaseIntermediateDirectory().ToString(), "Debug", DefaultTfm);
 
             var scoped = Path.Combine(intermediateOutputPath, "scopedcss", "Styles", "Pages", "Counter.rz.scp.css");
             new FileInfo(scoped).Should().Exist();
             new FileInfo(scoped).Should().Contain("b-overriden");
-            var generated = Path.Combine(intermediateOutputPath, "Razor", "Components", "Pages", "Counter.razor.g.cs");
-            new FileInfo(generated).Should().Exist();
-            new FileInfo(generated).Should().Contain("b-overriden");
```

Comment:
Could we figure out a way to preserve these checks with the compiler flag?

Reply:
Yeah, we'll need to do that once https://github.com/dotnet/roslyn/issues/51335 is resolved.